crates/biome_service/src/dome.rs

```diff
@@ -11,6 +11,15 @@ pub struct Dome {
 }
 
 impl Dome {
+    pub fn from_intern(intern_paths: &PathInternerSet) -> Self {
+        let guard = intern_paths.guard();
+        let mut paths = BTreeSet::new();
+        for path in intern_paths.iter(&guard) {
+            paths.insert(BiomePath::new(path));
+        }
+        Self { paths }
```

Comment:
The thing I don't understand about the `Dome` is this: We have a single set that our iterators pull from. Does `BiomePath` itself have an `Ord` implementation that defines ordering by priority? If so, I really think we should document that. If not, I wouldn't understand how this works unless it's by chance when you happen to pull paths in the right order.

Reply:
Yes, it has a `kind` field that defines the order. It isn't documented directly on `BiomePath` I think, and `kind` is documented. I can improve it for sure